<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rpgwindow</class>
 <widget class="QWidget" name="rpgwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1422</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1422</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1422</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Practice your skills!</string>
  </property>
  <widget class="QLabel" name="backgroundLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1422</width>
     <height>800</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="eatButton">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>650</y>
     <width>220</width>
     <height>130</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:enabled{
	
	background-color: rgba(172, 219, 255, 0);
	color: rgb(255, 255, 255);
}
QPushButton:!enabled {
	background-color: rgba(60,60,60,0);
	color:rgb(200,200,200);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Assets/Graphics/ActionButtons/eatButton.png</normaloff>:/Assets/Graphics/ActionButtons/eatButton.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>220</width>
     <height>220</height>
    </size>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backAwayButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>650</y>
     <width>220</width>
     <height>130</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:enabled{
	
	background-color: rgba(172, 219, 255, 0);
	color: rgb(255, 255, 255);
}
QPushButton:!enabled {
	background-color: rgba(60,60,60,0);
	color:rgb(200,200,200);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Assets/Graphics/ActionButtons/backAwayButton.png</normaloff>:/Assets/Graphics/ActionButtons/backAwayButton.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>220</width>
     <height>220</height>
    </size>
   </property>
   <property name="shortcut">
    <string>B</string>
   </property>
  </widget>
  <widget class="QPushButton" name="playDeadButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>650</y>
     <width>220</width>
     <height>130</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:enabled{
	
	background-color: rgba(172, 219, 255, 0);
	color: rgb(255, 255, 255);
}
QPushButton:!enabled {
	background-color: rgba(60,60,60,0);
	color:rgb(200,200,200);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Assets/Graphics/ActionButtons/playDeadButton.png</normaloff>:/Assets/Graphics/ActionButtons/playDeadButton.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>220</width>
     <height>220</height>
    </size>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </widget>
  <widget class="QPushButton" name="runAwayButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>650</y>
     <width>220</width>
     <height>130</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:enabled{
	
	background-color: rgba(172, 219, 255, 0);
	color: rgb(255, 255, 255);
}
QPushButton:!enabled {
	background-color: rgba(60,60,60,0);
	color:rgb(200,200,200);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Assets/Graphics/ActionButtons/runAwayButton.png</normaloff>:/Assets/Graphics/ActionButtons/runAwayButton.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>220</width>
     <height>220</height>
    </size>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </widget>
  <widget class="QPushButton" name="yellButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>650</y>
     <width>220</width>
     <height>130</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:enabled{
	
	background-color: rgba(172, 219, 255, 0);
	color: rgb(255, 255, 255);
}
QPushButton:!enabled {
	background-color: rgba(60,60,60,0);
	color:rgb(200,200,200);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Assets/Graphics/ActionButtons/yellButton.png</normaloff>:/Assets/Graphics/ActionButtons/yellButton.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>220</width>
     <height>220</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QPushButton" name="intimidateButton">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>650</y>
     <width>220</width>
     <height>130</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:enabled{
	
	background-color: rgba(172, 219, 255, 0);
	color: rgb(255, 255, 255);
}
QPushButton:!enabled {
	background-color: rgba(60,60,60,0);
	color:rgb(200,200,200);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Assets/Graphics/ActionButtons/intimidateButton.png</normaloff>:/Assets/Graphics/ActionButtons/intimidateButton.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>220</width>
     <height>220</height>
    </size>
   </property>
   <property name="shortcut">
    <string>I</string>
   </property>
  </widget>
  <widget class="QLabel" name="enemyLabel">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>200</y>
     <width>471</width>
     <height>281</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QProgressBar" name="healthBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>600</y>
     <width>591</width>
     <height>40</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="format">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bestiaryButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:enabled{
	background-color: rgba(0, 0, 0,0);
	color: rgb(255, 255, 255);
	border-color: rgb(255, 255, 255);
	border-image: url(:/Assets/Graphics/ActionButtons/BestiaryBorder.png);
}
QPushButton:!enabled {
	background-color: rgba(60,60,60,0);
	color:rgb(200,200,200);
	border-color: rgb(255, 255, 255);
}
QPushButton:released{
	border-image: url(:/Assets/Graphics/ActionButtons/BestiaryBorder.png);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	color: rgb(136, 136, 136);
	border-image: url(:/Assets/Graphics/ActionButtons/BestiaryBorder_P.png);
}</string>
   </property>
   <property name="text">
    <string>Bestiary</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logOutline">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>771</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Assets/Graphics/ActionButtons/Textbox.png</normaloff>:/Assets/Graphics/ActionButtons/Textbox.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>771</width>
     <height>500</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="healthBarLabel">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>600</y>
     <width>571</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>731</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="logLabel">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="defeatedLabel">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>10</y>
     <width>320</width>
     <height>52</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Assets/Graphics/ActionButtons/defeatedBox.png);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Enemies Defeated: 0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="hintButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:enabled{
	background-color: rgba(0, 0, 0,0);
	color: rgb(255, 255, 255);
	border-color: rgb(255, 255, 255);
	border-image: url(:/Assets/Graphics/ActionButtons/BestiaryBorder.png);
}
QPushButton:disabled {
	background-color: rgba(60,60,60,0);
	color:rgb(200,200,200);
	border-color: rgb(255, 255, 255);
}
QPushButton:released{
	border-image: url(:/Assets/Graphics/ActionButtons/BestiaryBorder.png);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	color: rgb(136, 136, 136);
	border-image: url(:/Assets/Graphics/ActionButtons/BestiaryBorder_P.png);
}</string>
   </property>
   <property name="text">
    <string>Hint</string>
   </property>
   <property name="shortcut">
    <string>T</string>
   </property>
  </widget>
  <widget class="QLabel" name="hintBackgroundLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Assets/Graphics/ActionButtons/BestiaryBorder_P.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>backgroundLabel</zorder>
  <zorder>hintBackgroundLabel</zorder>
  <zorder>eatButton</zorder>
  <zorder>backAwayButton</zorder>
  <zorder>playDeadButton</zorder>
  <zorder>runAwayButton</zorder>
  <zorder>yellButton</zorder>
  <zorder>intimidateButton</zorder>
  <zorder>enemyLabel</zorder>
  <zorder>healthBar</zorder>
  <zorder>bestiaryButton</zorder>
  <zorder>logOutline</zorder>
  <zorder>healthBarLabel</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>defeatedLabel</zorder>
  <zorder>hintButton</zorder>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
